src/helm/benchmark/data_overlap/light_scenario.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+try:
+    from scenarios.scenario import ScenarioSpec
+except Exception:
+    from helm.benchmark.scenarios.scenario import ScenarioSpec
+
+
+@dataclass(frozen=True)
+class LightInstance:
+    """
+    A lighter `Instance` with only text fields.
+    """
+
+    input: str
+    """The input"""
+
+    references: List[str]
+    """References that help us evaluate"""
+
+    id: Optional[str] = None
+    """Helm instance id"""
+
+
+@dataclass(frozen=True)
+class LightScenarioKey:
+    """
+    Key for LightScenario
+    """
+
+    scenario_spec: ScenarioSpec
+
+    split: str
+
+    def __eq__(self, other):
```

Comment:
Don't need this: the default eq and hash is correct because `ScenarioSpec` is frozen.

Reply:
Removed, thanks